### PR TITLE
Fixes Contract Test Errors for MetricFilter and Destination.

### PR DESCRIPTION
**Pull request by AaronLi**
*Issue #, if available:*
None
*Description of changes:*
 Fixes Contract Test Errors for MetricFilter and Destination.

Destination changes are to fix the `AssertionError: The deletion handler's response object MUST NOT contain a model` error

MetricFilter changes are to fix the same error as above as well as `AssertionError: All properties specified in the request MUST be present in the model returned, and they MUST match exactly, with the exception of properties defined as writeOnlyProperties in the resource schema
`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-logs-destination/src/main/java/software/amazon/logs/destination/DeleteHandler.java [hunk 1]
```diff
@@ -4,11 +4,7 @@
 import software.amazon.awssdk.services.cloudwatchlogs.model.CloudWatchLogsException;
 import software.amazon.awssdk.services.cloudwatchlogs.model.DeleteDestinationRequest;
 import software.amazon.awssdk.services.cloudwatchlogs.model.DeleteDestinationResponse;
-import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
-import software.amazon.cloudformation.proxy.Logger;
-import software.amazon.cloudformation.proxy.ProgressEvent;
-import software.amazon.cloudformation.proxy.ProxyClient;
-import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.cloudformation.proxy.*;
```
Comment (sfbaker7) [inline aws-logs-destination/src/main/java/software/amazon/logs/destination/DeleteHandler.java:7]: nit: remove wild card

File: aws-logs-metricfilter/src/main/java/software/amazon/logs/metricfilter/CreateHandler.java [hunk 2]
```diff
@@ -5,12 +5,7 @@
 import software.amazon.awssdk.services.cloudwatchlogs.model.*;
 import software.amazon.awssdk.services.cloudwatchlogs.model.ResourceNotFoundException;
 import software.amazon.cloudformation.exceptions.*;
-import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
-import software.amazon.cloudformation.proxy.HandlerErrorCode;
-import software.amazon.cloudformation.proxy.Logger;
-import software.amazon.cloudformation.proxy.ProgressEvent;
-import software.amazon.cloudformation.proxy.ProxyClient;
-import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.cloudformation.proxy.*;
```
Comment (sfbaker7) [inline aws-logs-metricfilter/src/main/java/software/amazon/logs/metricfilter/CreateHandler.java:8]: same comment

**Comment by markkuhn** [comment]
This PR has been open for more than a year. Can we close it?

**Comment by shri-tambe** [comment]
Let's fix the conflicts and rebase against latest changes.

**Pull request closed without merging**